scripts/travis/cleanup.js

```diff
@@ -84,20 +56,40 @@ const getCurrentRevs = (db, response) => {
   return db.allDocs({ keys: ids });
 };
 
+const queryReleases = (db, options) => {
+  return db.query('builds/releases', options)
+    .then(response => getCurrentRevs(db, response));
+};
+
+const queryReleasesByDate = (db, daysToKeep) => {
+  return queryReleases(db, {
+    startkey: [ 'branch', 'medic', 'medic' ],
+    endkey: [ 'branch', 'medic', 'medic', getEndDate(daysToKeep) ],
+    limit: MAX_BUILDS_TO_DELETE
+  });
+};
+
 const testingBuilds = () => {
-  return getTestingBuilds(testingDb)
+  console.log('Querying for old testing builds...');
+  return queryReleasesByDate(testingDb, DAYS_TO_KEEP_TEST)
     .then(response => remove(testingDb, response));
 };
 
 const branchBuilds = () => {
-  return getBranchBuilds(stagingDb)
-    .then(response => getCurrentRevs(stagingDb, response))
+  console.log('Querying for old branch builds...');
+  return queryReleasesByDate(stagingDb, DAYS_TO_KEEP_BRANCH)
```

Comment:
don't we also need to get the revs here? we're not longer getting them implicitly, as `queryReleasesByDate` doesn't request them.